docs/content/configuration/index.md

```diff
@@ -75,6 +75,22 @@ The following path is used for service discovery. It is **not** affected by `dru
 |--------|-----------|-------|
 |`druid.discovery.curator.path`|Services announce themselves under this ZooKeeper path.|`/druid/discovery`|
 
+### Exhibitor
+
+[Exhibitor](https://github.com/Netflix/exhibitor/wiki) is a supervisor system for ZooKeeper.
+Exhibitor can dynamically scale-up/down the cluster of ZooKeeper servers.
+Druid can update self-owned list of ZooKeeper servers through Exhibitor without restarting.
+That is, it allows Druid to keep the connections of Exhibitor-supervised ZooKeeper servers.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.exhibitor.service.hosts`|Comma-separated hostnames of Exhibitor instances. Please specify this property instead of `druid.zk.service.host` if you want to use Exhibitor-supervised cluster.|none|
+|`druid.exhibitor.service.port`|The REST port used to connect to Exhibitor.|`8080`|
```

Comment:
This port is default port of Exhibitor server.
On the client side (on Druid nodes), it will be used another free port for HTTP Client.

Exhibitor-server’s port is specified by the command line options:
```
--port <arg>    Port for the HTTP Server. Default is: 8080
```
https://github.com/Netflix/exhibitor/wiki/Running-Exhibitor